tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34
```

Comment:
I don't think python3 ever worked. I just added it so that we could test it over time, but we're pretty far from it at the moment since we don't use any compat libraries like `six`